### DISCUSSION TITLE
Update to LES/WMLES

### DISCUSSION THREAD

**Discussion by GomerOfDoom** [su2code/SU2#1622] [General]
Hello all,

I am beginning an effort to implement/update the LES/WMLES capabilities in SU2.

Currently, my understanding is that the LES/WMLES for the FV side (as opposed to the high-order DG-FEM side) of SU2 was originally implemented by Eduardo Molina in the branch "feature_WallModelLES" and that Eduardo then tried to update the capability in branch "feature_WMLES-Dev." However, from my conversations with Eduardo, and looking at those branches, I believe that the LES/WMLES was never merged into develop. Is this correct?

Also based on a conversation with Eduardo, we felt the best way to merge the LES/WMLES implementation into develop would be to create a new branch from the most up-to-date version of develop and start there, rather than try to merge all of the changes that have been made to develop back into those older branches. Thus, I have created branch "feature_WMLES_update" and will use that as the foundation on which to re-implement the LES/WMLES code.

I intend to do the re-implementation in stages, starting with a very simple case using no wall-model and no SGS model (i.e. "implicit LES"). As a test/debug case, I was going to use a periodic channel (Poiseuille), but it's occurring to me that the Taylor-Green vortex case (or some other case) might be better. Does anyone have any thoughts on this?

Anyway, I wanted to start a running thread/discussion about the re-implementation of LES/WMLES here. Please chime in with any/all thoughts.

Cheers,
Paul

**Comment by pcarruscag** [comment]
Hi Paul,
That work by Eduardo was indeed not merged, it is in PR #1120 which I reopened after our convo in the dev meeting, thinking you'd start from there.
I think what you propose it the best way. Please open a draft PR for your new branch once it starts having code. That way we can talk about implementation decisions directly over code.
Cheers,
Pedro

**Comment by rois1995** [comment]
Hi there, are there any updates on this matter? I have checked the branch and the last changes are dated to one year ago.
